src/Raven.Studio/typescript/viewmodels/database/query/query.ts

```diff
@@ -64,7 +64,22 @@ class highlightSection {
 }
 
 class timeSeriesTableDetails {
+    
+    tabText: KnockoutComputed<string>;
+    tabInfo: KnockoutComputed<string>;
+    
     constructor(public documentId: string, public name: string, public value: timeSeriesQueryResultDto) {
+
+        this.tabText = ko.pureComputed(() => {
+            return `TS - ${generalUtils.truncateDocumentId(this.documentId)}`;
```

Comment:
do we need to escape that or it is rendered using text data-bind?

Reply:
tabText is rendered with `data-bind="text: tabText"`

Reply:
ok then we don't need to escape